tests/js/panelsTests.js

```diff
@@ -310,6 +347,29 @@ https://github.com/gpii/universal/LICENSE.txt
         jqUnit.assertEquals("The model value for the \"selectedLang\" is set correctly to " + expected, expected, that.model.selectedLang);
     };
 
+    gpii.tests.langTester.verifyDisplayLangIndex = function (that, expected) {
+        jqUnit.assertEquals("The model value for displayLangIndex is set to " + expected, expected, that.model.displayLangIndex);
+    };
+
+    gpii.tests.langTester.getFirstButtonTop = function (that) {
+        return that.locate("langRow").offset().top;
+    };
+
+    gpii.tests.langTester.verifyInitDisplayedLang = function (that, expected) {
+        that.prevFirstButtonTop = gpii.tests.langTester.getFirstButtonTop(that);
```

Comment:
I"m not really sure why this is necessary, maybe a comment here would be helpful.